dagger-core/src/test/java/io/odpf/dagger/core/config/KafkaEnvironmentVariablesTest.java

```diff
@@ -14,6 +14,7 @@
     @Test
     public void shouldReturnKafkaEnvVarsPositive() {
         Configuration configuration = new Configuration();
+        //TODO remove hashmap and directly set the config
```

Comment:
ahh. we shouldn't commit the TODO comments.

Reply:
Yes, let's first merge all refactor tests into a separate branch then merge to main instead of all of them going into main.

Reply:
sounds good

Reply:
just fyi , this todo has been resolved here https://github.com/odpf/dagger/pull/66/files#diff-07e3111b5f5522801765e8324fa3f6c287b02828d08187670875ca574a9b49f7